myft-hint/main.scss

```diff
@@ -54,7 +53,7 @@
 	}
 
 	.myft-hint__content {
-		@include oTypographyBody;
+		@include oFontsInclude(MetricWeb, $weight: semibold);
```

Comment:
This doesn’t do what you’re probably going for. `oFontsInclude` only declares the name/url of the font, rather than tell `.myft-hint__content` to use it. It doesn’t affect the visuals of your hint at all, although the _removal_ of `oTypographyBody` does.

Options:
- if you’re happy with the visuals as is (I haven’t seen it), you can remove this line of code
- if you wanted it to be bold, you can replace this line with `@include oTypographySansBold('m')`, but that’ll knock your font-size up a touch ([for reference](https://github.com/Financial-Times/o-typography#sans))

Gimme a shout if I’m chatting nonsense, which seems likely